lib/taskana-core/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java

```diff
@@ -590,20 +725,37 @@ void should_OnlyClaimTasksWhichAreNotClaimed_When_BulkForceCompletingTasks() thr
     assertThat(results.containsErrors()).isFalse();
 
     task = taskService.getTask(id1);
+
     assertThat(task.getState()).isEqualTo(TaskState.COMPLETED);
     // do not update claimed timestamp for already claimed task
     assertThat(task.getClaimed()).isBefore(beforeBulkComplete);
     assertThat(task.getCompleted())
         .isEqualTo(task.getModified())
         .isAfterOrEqualTo(beforeBulkComplete);
-    assertThat(task.getOwner()).isEqualTo("user-b-2");
+    assertThat(task.getOwner()).isEqualTo("user-1-1");
 
     task = taskService.getTask(id2);
+
     assertThat(task.getState()).isEqualTo(TaskState.COMPLETED);
     assertThat(task.getCompleted())
         .isEqualTo(task.getClaimed())
         .isEqualTo(task.getModified())
         .isAfterOrEqualTo(beforeBulkComplete);
-    assertThat(task.getOwner()).isEqualTo("user-b-2");
+    assertThat(task.getOwner()).isEqualTo("user-1-1");
+  }
+
+  private TaskBuilder createTaskClaimedByUser() {
+    return createDefaultTask()
+        .owner("user-1-1")
+        .created(Instant.parse("2018-01-29T15:55:00Z"))
+        .state(TaskState.CLAIMED)
```

Comment:
when we want to use our API correctly we have to set the claimed timestamp aswell :)